lib/capybara/webkit/driver.rb

```diff
@@ -287,7 +287,8 @@ def clear_cookies
     end
 
     def invalid_element_errors
-      [Capybara::Webkit::ClickFailed]
+      [Capybara::Webkit::ClickFailed,
+       Capybara::Webkit::NodeNotAttachedError]
```

Comment:
@jferris Yes ---  It happens to be treated by the main synchronize loop in Capybara as an invalid element error because it's derived from Capybara::ElementNotFound, but it really should be returned in #invalid_element_errors which a fixing of some tests in Capybara showed.